### PR TITLE
Added new chart for demoing with hardcoded data

### PR DESCRIPTION
**Pull request by WGNeff11**

### CODE REVIEW

File: src/app/components/line-chart/line-chart.component.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<div>
+    
+    <!-- <button class="btn btn-outline-dark rounded-0" (click)="showRoute1()">Route 1</button>
```
Comment (IkenoXamos) [inline src/app/components/line-chart/line-chart.component.html:3]: Let's endeavor to not include any commented out code in the codebase. It just keeps everything clean.

File: src/app/components/line-chart/line-chart.component.ts [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+  animations: boolean = true;
+  xAxis: boolean = true;
+  yAxis: boolean = true;
+  showYAxisLabel: boolean = true;
+  showXAxisLabel: boolean = true;
+  xAxisLabel: string = 'Request';
+  yAxisLabel: string = 'Time (Miliseconds)';
+  timeline: boolean = true;
+  selectedRoute;
+  colorScheme = {
```
Comment (IkenoXamos) [inline src/app/components/line-chart/line-chart.component.ts:29]: Can this be moved into CSS?

File: src/app/components/line-chart/line-chart.component.ts [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+  onDeactivate(data): void {
+    console.log('Deactivate', JSON.parse(JSON.stringify(data)));
+  }
+
+
+  showRoute(routeName) {
+    this.selectedRoute = this.multiOne[routeName];
+  }
+
+  // showRoute2() {
```
Comment (IkenoXamos) [inline src/app/components/line-chart/line-chart.component.ts:233]: Commented out code could be removed.

File: src/app/components/navbar/navbar.component.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<nav class="navbar bg-dark fixed-top bg-light custom">
```
Comment (IkenoXamos) [inline src/app/components/navbar/navbar.component.html:1]: Why was this needed?

File: src/app/components/line-chart/line-chart.component.ts [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+  timeline: boolean = true;
+  selectedRoute;
+  colorScheme = {
+    domain: ['#5AA454', '#E44D25', '#CFC0BB', '#7aa3e5', '#a8385d', '#aae3f5']
+  };
+
+  constructor() {
+    this.routeNames = new Array();
+    this.multiOne =
+    {
```
Comment (IkenoXamos) [inline src/app/components/line-chart/line-chart.component.ts:36]: Perhaps this could be imported from the data file?

File: src/app/data.ts [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+          },
+
+        ]
+      }
+    ]
+  }
+]
+;
+
+// export var multi = [
```
Comment (IkenoXamos) [inline src/app/data.ts:155]: Commented out code

**Review by IkenoXamos — CHANGES_REQUESTED** [review]
Looks pretty good, but some minor changes needed.

File: src/app/components/line-chart/line-chart.component.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
+<div>
+    <!-- <div *ngFor="let routeName of routeNames">
```
Comment (IkenoXamos) [inline src/app/components/line-chart/line-chart.component.html:2]: Please remove

File: src/app/components/line-chart/line-chart.component.ts (hunk 2 above)
Comment (WGNeff11) [reply src/app/components/line-chart/line-chart.component.ts:30]: This is the default and most common way to provide color configuration to the chart, from preliminary research it seems CSS is not possible.

**Review by IkenoXamos — CHANGES_REQUESTED** [review]
One last comment of code that needs to be cleaned up

File: src/app/components/navbar/navbar.component.html (hunk 4 above)
Comment (WGNeff11) [reply src/app/components/navbar/navbar.component.html:1]: Having the navbar as fixed-top made our content begin at the top of the page and underneath the navbar (therefore hiding it from view), rather than at the base of it.

**Review by IkenoXamos — APPROVED** [review]
Looks great!
